src/projects/shared/api/circuitdesigner/src/public/impl/Viewport.ts

```diff
@@ -245,6 +245,10 @@ export class ViewportImpl<T extends CircuitTypes> extends MultiObservable<Viewpo
     public get debugOptions(): DebugOptions {
         return this.state.debugOptions;
     }
+    public setRenderOptions(options: Partial<Pick<RenderOptions, "showGrid">>): void {
+        this.state.circuitState.renderOptions = { ...this.state.circuitState.renderOptions, ...options };
```

Comment:
does this not break anything? I'm worried it would remove the methods that RenderOptions has one it, especially since they involve `this.`